install/kubernetes/cilium/values.yaml

```
@@ -1443,6 +1443,9 @@ tls:
 #   - geneve
 tunnel: "vxlan"
 
+# -- Configure the underlying network MTU to overwrite auto-detected MTU.
+configuredMTU: ""
```

All Helm options are configured options, any reason not to just define this as MTU?

```suggestion
MTU: ""
```

(Or we could argue that MTU should be nested under datapath instead of being directly at the top level configuration for Cilium, I don't have a strong opinion either way).

---

it is a name I steal from inside the code, I thought about name `MTU` but   I knew someone like you  is going to suggest a better name :-).